enterprise/backend/test/metabase_enterprise/audit_app/permissions_test.clj

```diff
@@ -86,7 +86,7 @@
 
           (testing "Users without access to the audit collection cannot run any queries on the audit DB, even if they
                    have data perms for the audit DB"
-            (binding [api/*current-user-permissions-set* (delay #{(perms/data-perms-path perms/audit-db-id)})]
+            (binding [api/*current-user-permissions-set* (delay #{})]
```

Comment:
good call, fixed using `with-full-data-perms-for-all-users` and `with-test-user`. we still need to override the permissions set for now because we explicitly don't want the user to have collection perms